hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/OzoneRepair.java

```diff
@@ -22,8 +22,11 @@
 import org.apache.hadoop.hdds.cli.GenericCli;
 import org.apache.hadoop.hdds.cli.HddsVersionProvider;
 import org.apache.hadoop.hdds.conf.OzoneConfiguration;
+import org.apache.hadoop.hdds.tracing.TracingUtil;
 import picocli.CommandLine;
 
+import static org.apache.hadoop.fs.ozone.Constants.OZONE_DEFAULT_USER;
```

Comment:
This is just used to create a default user directory in the Ozone filesystem. It is not related to the user the OM or any other component is run as, although "hdfs" does happen to be a common choice for that.

Reply:
makes sense! removed!